playbooks/roles/vector/tasks/main.yml

```diff
@@ -0,0 +1,80 @@
+---
+#
+# vector
+#
+# Overview:
+#
+# Installs vector
+##
+# Dependencies:
+#
+# Example play:
+#   roles:
+#   - vector
+#
+- name: "Prepare the vector package for apt based in https://vector.dev/docs/setup/installation/package-managers/apt/"
+  shell: |
+    curl -1sLf 'https://repositories.timber.io/public/vector/cfg/setup/bash.deb.sh' |  sudo -E bash
```

Comment:
This is a very broad execution. What happens if this file is deleted or has an error? (worse yet, what if this file is malicious)

I would use https://docs.ansible.com/ansible/latest/collections/ansible/builtin/apt_repository_module.html to accomplish the same